source/renderer/zettlr-attachments.js

```diff
@@ -106,6 +106,10 @@ class ZettlrAttachments {
         if (FILETYPES_IMG.includes(a.ext.toLowerCase())) {
           // Override the drag data with a link to the image
           let uri = decodeURIComponent(a.path)
+
+          // Make the link relative instead of absolute
+          uri = path.basename(uri)
```

Comment:
This won't work because people can have a totally different directory selected, so it needs to be something like (I think) `path.relative(markdownBaseLink, uri)`

Soooo it does make sense to call your new function here, and not mingle with the absolute path here, as this is in the drag-data anyway. This will break in a lot of circumstances!